scripts/test-utils.js

```diff
@@ -1,16 +1,126 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Can we add this script to https://github.com/cloudscape-design/build-tools?

Reply:
Or maybe the test-utils core?

Reply:
You mean pluralization or the file header?